pkg/platforms/kubernetes/client_alphav2_linux.go

```diff
@@ -165,3 +173,25 @@ func getNS(origctx context.Context, cli criapi.RuntimeService, ns string, contai
 	inod, err := strconv.ParseInt(matches[1], 10, 0)
 	return uint64(inod), err
 }
+
+func getEnv(cli criapi.RuntimeService, containerid string) (map[string]string, error) {
+	cmd := []string{"printenv"}
+	envVariables := make(map[string]string)
+	stdout, _, err := cli.ExecSync(containerid, cmd, time.Second)
+	if err != nil {
+		log.WithField("err", err).Warn("Error exec sync to get environment variables!")
+		return envVariables, err
+	}
+	vars := strings.Split(string(stdout), "\n")
+	for _, line := range vars {
+		if strings.HasPrefix(line, "JAVA_") || strings.HasPrefix(line, "GO_") {
```

Comment:
I am not aware of a need for the `GO_` prefix env vars or any other `JAVA_` prefixed env vars so I would prefer not to gather them.

If we do need to call this during runtime, I would prefer using `echo $JAVA_TOOL_OPTIONS` instead of `printenv`

Reply:
For go we have env GODEBUG=gctrace=1,schedtrace=1000 godoc -http=:8080, we also have something like JAVA_OPTS,  CATALINA_OPTS . There is more details at https://stackoverflow.com/questions/28327620/difference-between-java-options-java-tool-options-and-java-opts

Reply:
It's not clear to me how those values are useful

Reply:
All those variables are alternate ways of providing debug port.

Reply:
are we good with conversation, can we mark it resolved? Any todo here?